contracts/helpers/AssetsManagerMock.sol

```diff
@@ -26,56 +26,66 @@ contract AssetsManagerMock is AssetsManagerInterface {
     }
 
 
-    function isAssetSymbolExists(bytes32 _symbol) constant returns (bool) {
+    function isAssetSymbolExists(bytes32 _symbol) public constant returns (bool) {
         return assets[_symbol] != 0x0;
     }
 
-    function getAssetsSymbols() constant returns (bytes32[]) {
+    function getAssetsSymbols() public constant returns (bytes32[]) {
         return symbols;
     }
 
-    function getAssetsSymbolsCount() constant returns (uint) {
+    function getAssetsSymbolsCount() public constant returns (uint) {
         return symbols.length;
     }
 
-    function getAssetBySymbol(bytes32 symbol) constant returns (address) {
+    function getAssetBySymbol(bytes32 symbol) public constant returns (address) {
         return assets[symbol];
     }
 
-    function addAsset(address asset, bytes32 _symbol, address owner) returns (bool) {
-        if (assets[_symbol] == 0) {
+    function addAsset(address asset, bytes32 _symbol) public returns (bool) {
+        if (assets[_symbol] == 0x0) {
             symbols.push(_symbol);
             assets[_symbol] = asset;
             return true;
         }
         return false;
     }
 
-    function() {
-        throw;
+    function() public {
+        revert();
     }
 
-    function getAssetsForOwner(address platform, address owner) constant returns (bytes32[]) {
+    function getAssetsForOwner(address platform, address owner) public constant returns (bytes32[]) {
```

Comment:
Arguments, that are not used could be skipped, f.e. in this case it would be `function getAssetsForOwner(address, address) public constant returns (bytes32[])`

Reply:
Ok, I've fixed it.